p2p/src/net/default_backend/default_networking_service.rs

```diff
@@ -41,7 +41,7 @@ use super::{backend::Backend, ConnectivityHandle, MessagingHandle, SyncingEventR
 // The preferred protocol version.
 // Note that we intentionally keep this constant private, because most of the code should
 // not depend on its value.
-const PREFERRED_PROTOCOL_VERSION: SupportedProtocolVersion = SupportedProtocolVersion::V1;
+const PREFERRED_PROTOCOL_VERSION: SupportedProtocolVersion = SupportedProtocolVersion::V2;
```

Comment:
I would merge this PR... but upgrading this to V2 without the fix can be an issue.